rnd/autogpt_server/autogpt_server/cli.py

```diff
@@ -0,0 +1,109 @@
+"""
+The command line interface for the agent server
+"""
+
+from multiprocessing import freeze_support
+from multiprocessing.spawn import freeze_support as freeze_support_spawn
+
+import click
+
+
+@click.group()
+def main():
+    """AutoGPT Server CLI Tool"""
+
+
+@main.command()
+def background() -> None:
```

Comment:
I guess up to this point, there is no such thing as "running in the background" ?
The background part only happens on `run` when you call Popen

Reply:
That is correct, however, the purpose of the function is to be used to run the system in the background.

We could call this one `start` and the rename the `start` function to `start-in-background`. Though I think the semantics this way round is nicer.